src/System.Private.CoreLib/src/System/Exception.CoreCLR.cs

```diff
@@ -443,7 +443,6 @@ internal void SetCurrentStackTrace()
             // when it's retrieved.
             var sb = new StringBuilder(256);
             new StackTrace(fNeedFileInfo: true).ToString(System.Diagnostics.StackTrace.TraceFormat.TrailingNewLine, sb);
-            sb.AppendLine(SR.Exception_EndStackTraceFromPreviousThrow);
```

Comment:
I removed this line (and thus the string) because elsewhere where remote stack trace and stack trace are combined, they are concatenated without a separator. Eg., line 43.

Reply:
Please put it back. Have you looked at what the stack traces look like without it?